integration/src/main/java/scripts/utils/ClientTestUtils.java

```diff
@@ -329,6 +331,19 @@ public static <T> T getWithRetryOnException(
     return result;
   }
 
+  public static void getWithRetryOnFalse(SupplierWithException<Boolean> function) throws Exception {
+    getWithRetryOnException(function);
```

Comment:
Good catch - I was supposed to be calling `tryBooleanFunction` which converts a false into an exception.
But rereading the code, it really doesn't work right. If the boolean function throws, then we retry instead of
allowing the exception. It is unused code, so I'll delete it and code a different way when I get back to RemoveUser.